cob_description/urdf/sensors/sick_s300_laser.gazebo.xacro

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0"?>
 <robot xmlns:xacro="http://www.ros.org/wiki/xacro">
 
-  <xacro:macro name="sick_s300_laser_gazebo_v0" params="name ros_topic update_rate min_angle max_angle">
+  <xacro:macro name="sick_s300_laser_gazebo_v0" params="name">
     <gazebo reference="${name}_link">
       <sensor name="${name}" type="ray">
         <always_on>true</always_on>
-        <update_rate>${update_rate}</update_rate>
+        <update_rate>12.0</update_rate>
```

Comment:
e.g. [here](https://github.com/ipa320/cob_robots/blob/indigo_dev/cob_hardware_config/cob3-2/config/laser_front.yaml#L7) is a parameter to change the update rate. It works okay, dropping messages if there are too many, but it is the same everywhere. Thus, we could leave the `${update_rate}` here, to allow changing depending on the robot configuration.

But from my point of view, the way it is done here is fine.